src/garage/experiment/local_runner.py

```diff
@@ -165,7 +174,7 @@ def restore(self, from_dir, from_epoch='last'):
                 Not applicable when snapshot_mode='last'.
 
         Returns:
-            A SimpleNamespace for train()'s arguments.
+            SimpleNamespace: A SimpleNamespace for train()'s arguments.
```

Comment:
this type doesn't tell the user anything about the interface, which means it should probably be different.

in python this kind of type is often a namedtuple:
```python
import collections

TrainArgs = collections.namedtuple('TrainArgs', [
    'n_epochs',
    'n_epoch_cycles',
    'batch_size',
    'plot',
    'store_paths',
    'pause_for_plot',
    'start_epoch'])
```

For some reason when this was initially implemented we used `SimpleNamespace` instead of `namedtuple`. I'm not sure why. If you still *really* need `SimpleNamespace` then you can at least create a type by doing this:
```python
import types

class TrainArgs(types.SimpleNamespace):
    pass
```

Unfortunately this gives the user no information on what is in a `TrainArgs` but at least it has its own type.